ingeniamotion/fsoe_master/handler.py

```diff
@@ -88,6 +89,10 @@ def __init__(
         # Parameters that will be transmitted during the fsoe parameter state
         fsoe_application_parameters: list[FSoEApplicationParameter] = []
 
+        # Set MDP module
+        self.__set_configured_module_ident_1(use_sra=use_sra)
+
+        safety_module = self.__get_safety_module()
```

Comment:
Instead of writing and reading it back, maybe __set_configured_module_ident_1 could already return the module it has found and set.